.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,80 @@
+name: "🐛 Bug Report"
+description: Create a new ticket for a bug.
+title: "🐛 [BUG] - <title>"
+labels: [
+  "bug"
+]
+body:
+  - type: textarea
```

Comment:
I think there's something slightly wrong with the formatting. I'm seeing the following error when I try to view the file:
<img width="389" alt="image" src="https://user-images.githubusercontent.com/57045550/194024373-215105dc-919f-4e4d-8a76-68482c5c8bc7.png">

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms